frontend/src/app/shared/components/autocompleter/op-autocompleter/op-autocompleter.component.ts

```diff
@@ -228,6 +228,8 @@ export class OpAutocompleterComponent<T extends IAutocompleteItem = IAutocomplet
 
   @Input() public relations?:boolean = false;
 
+  @Input() public debounceTimeMs:number = 250;
+
```

Comment:
Do you think you could make it configurable for feature tests? like make it 10 ms so that feature tests do not wait for the debounce?